oresat_configs/base/dxwifi.yaml

```diff
@@ -16,6 +16,18 @@ objects:
         access_type: ro
         unit: C
 
+  - index: 0x4002
+    name: live_status
```

Comment:
No need to make a new obj, use the existing status obj

Reply:
Done